src/core_modules/capture-core/components/Pages/Search/SearchPage.component.js

```diff
@@ -150,7 +152,7 @@ const Index = ({
                             data-test="dhis2-capture-general-purpose-error-mesage"
                             className={classes.generalPurposeErrorMessage}
                         >
-                            {generalPurposeErrorMessage}
+                            {i18n.t(generalPurposeErrorMessage)}
```

Comment:
This won't work, you need to use i18n.t where the actual text is defined.

Reply:
Dont see the reason to have this stored in redux. I removed it from there.